api.py

```diff
@@ -1,11 +1,97 @@
-from flask import Flask
+import json
+import os
+import urllib
+from json import JSONDecodeError
 
-from app.views.status import status_blueprint
-from app.views.validate import validate_blueprint
+import requests
+import uvicorn
+from fastapi import Body, FastAPI, Response
+from requests import RequestException
+from structlog import get_logger
+
+from app.validators.questionnaire_validator import QuestionnaireValidator
+
+app = FastAPI()
+
+
+@app.get("/status")
+async def status():
+    return Response(status_code=200)
+
+
+logger = get_logger()
+
+AJV_HOST = os.getenv("AJV_HOST", "localhost")
+
+AJV_VALIDATOR_URL = f"http://{AJV_HOST}:5002/validate"
+
+
+@app.post("/validate")
+async def validate_schema_request_body(payload=Body(None)):
+    logger.info("Validating schema")
+    return await validate_schema(payload)
+
+
+@app.get("/validate")
```

Comment:
We should probably document this in the readme, how this works, as not sure it recorded anywhere. Not necessarily as part of this change though 👍

Reply:
I can try writing something, better than having nothing like now.